src/env-inl.h

```diff
@@ -513,6 +520,12 @@ inline void Environment::SetTemplateMethod(v8::Local<v8::FunctionTemplate> that,
   function->SetName(name_string);  // NODE_SET_METHOD() compatibility.
 }
 
+inline v8::Local<v8::Object> Environment::NewInternalFieldObject() {
+  v8::MaybeLocal<v8::Object> m_obj =
+      generic_internal_field_template()->NewInstance(this->context());
+  return m_obj.ToLocalChecked();
```

Comment:
I thought we need to check if the MayBe objects are empty before using them. What does ToLocalChecked do? Couldn't find much info on v8 docs.